_topics/en/cve.md

```diff
@@ -0,0 +1,186 @@
+        exist by specially crafting a real 64-byte transaction that gets
+        confirmed in a block.  Can be used to steal from SPV clients,
+        although other known attacks against SPV clients are cheaper.
+        Proposed to be fixed in the [consensus cleanup][topic consensus
+        cleanup] soft fork.
+
+    CVE-2017-18350:
+      link: /en/newsletters/2019/11/13/#cve-2017-18350-socks-proxy-vulnerability
+      summary: >
+        A vulnerability in Bitcoin Core affecting users of SOCKS
+        proxies.  Fixed in Bitcoin Core 0.16.0.
+
+    CVE-2013-2292:
+      link: https://bitcointalk.org/?topic=140078
+      summary: >
+        A vulnerability in Bitcoin Core that it was believed could've
+        allowed an attacker to create a block that would take over three
+        minutes to verify.  Proposed to be fixed in the [consensus
+        cleanup][topic consensus cleanup] soft fork, although the
+        suggestion solution encountered controversy.
+
+    CVE-2012-2459:
+      link: https://bitcointalk.org/?topic=102395
+      summary: >
+        It's possible to mutate a valid block into an invalid block that
+        commits to the same merkle root.  When vulnerable versions of
+        Bitcoin Core saw such an invalid block, they cached its
+        rejection and so would later reject the valid version of the
+        block, which could easily lead to short-term network partitions
+        where users could be tricked into accepting invalid bitcoins.
+        Fixed in Bitcoin 0.6.1.
+
+    CVE-2021-31876:
+      link: /en/newsletters/2021/05/12/#cve-2021-31876-discrepancy-between-bip125-and-bitcoin-core-implementation
+      summary: >
+        The BIP125 specification of opt-in Replace By Fee (RBF) says
+        that an unconfirmed parent transaction that signals
+        replaceability makes any child transactions that spend the
+        parent’s outputs also replaceable through inferred inheritance.
+        Bitcoin Core does not implement this behavior; Btcd does
+        implement it.
+
+---
+Some CVEs that have been frequently mentioned by Optech have their own
+pages in the Optech topic index (see this page's *see also* section for
+links).  For any other CVEs affecting Bitcoin and Lightning, this
+catch-all page collects any significant mentions of those CVEs in Optech
+content.  This pages's CVE's include:
+
+{% for alias in page.aliases %}
+  * **[{{alias}}]({{page.extra.cves[alias].link | default: "#CVE-LINK-NOT-PROVIDED"}}):** {{page.extra.cves[alias].summary}}
+
+  {%- for mention in page.optech_mentions -%}
+    {%- if mention.title contains alias %}
+      * [{{mention.title}}]({{mention.url}})
+    {%- endif -%}
+  {%- endfor -%}<br><br>
+
+{% endfor %}
+
+Please note that CVE entries may be removed from this page if we end up
+creating a new topic page specifically for that CVE.  Any moved CVEs
+should be listed in the *see also* section below.
```

Comment:
What do you think about also listing them here, with just a link to their own topic page? That's slightly more convenient for readers than having to look in two places (and would also allow you to remove this note and the similar "Some CVEs that have been frequently mentioned by Optech have their own pages in the Optech topic index (see this page's *see also* section for
links)." note at the top of the page.